lib/rademade_admin/model/configuration/csv_fields.rb

```diff
@@ -0,0 +1,12 @@
+# -*- encoding : utf-8 -*-
+require 'rademade_admin/model/configuration/fields'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.